fix(i18n): pages/home/home.tsx: move hardcoded strings

kuhnchris opened the pull request:
Moving the strings to the lang/ files.

While we're at it: Should we add a .github/ pull request template that users have to fill out when trying to file a PR.
Also, how is the progress for adding a change like this? Shall I open a PR for the translation/ repository aswell with the changes?

Right now we'd need to add these strings to the `external/lang/en.json` file
```
+++ b/en.json
@@ -466,7 +466,12 @@
                 "token_reveal": "{{name}}'s Token",
                 "onboarding": {
                     "welcome": "Welcome to",
-                    "pick": "Pick a username that you want people to be able to find you by, this can be changed later in settings."
+                    "pick": "Pick a username that you want people to be able to find you by, this can be changed later in settings.",
+                    "createGroup": "Create a group",
+                    "createGroupDescription": " of your friends, some cool bots, and throw a big party!",
+                    "testerServerDescription": "You can report issues and discuss improvements with us directly here.",
+                    "feedbackDescription": "Let us know how we can improve our app by giving us feedback.",
+                    "donateDescription": "Support the project by donating - thank you!"
                 },
                 "prompt": {
                     "confirm_leave": "Leave {{name}}?",

```

Let me know if I should move from camelCase to `create_group` or `create-group` or whatever. :-)

insertish commented:
> While we're at it: Should we add a .github/ pull request template that users have to fill out when trying to file a PR.

That would be nice, would probably include checkboxes for agreeing to code of conduct, for whether the code was tested and potentially a section for including screenshots if applicable, since those are often missed.

> Also, how is the progress for adding a change like this? Shall I open a PR for the translation/ repository aswell with the changes?

Yes, it needs to be changed there then pulled into this repository, not necessarily in the PR since I regularly bump language definitions and it may cause a conflict that way. I added a note to the [contributing guide](https://github.com/revoltchat/revolt/discussions/282#discussion-3777997).

> Let me know if I should move from camelCase to `create_group` or `create-group` or whatever. :-)

Would prefer under_scores.

kuhnchris commented:
> > While we're at it: Should we add a .github/ pull request template that users have to fill out when trying to file a PR.
> 
> That would be nice, would probably include checkboxes for agreeing to code of conduct, for whether the code was tested and potentially a section for including screenshots if applicable, since those are often missed.
Opened a 2nd PR for this #490 

> 
> > Also, how is the progress for adding a change like this? Shall I open a PR for the translation/ repository aswell with the changes?
> 
> Yes, it needs to be changed there then pulled into this repository, not necessarily in the PR since I regularly bump language definitions and it may cause a conflict that way. I added a note to the [contributing guide](https://github.com/revoltchat/revolt/discussions/282#discussion-3777997).

I added a PR on the translation repository. (https://github.com/revoltchat/translations/pull/24)

> > Let me know if I should move from camelCase to `create_group` or `create-group` or whatever. :-)
> 
> Would prefer under_scores.

Changed
---
Thanks for checking. :+1:

insertish commented:
Superseded by 599aab9a315f40a90571d8f7e4a879f676346314
